plugins/uefi-dbx/fu-uefi-dbx-device.c

```diff
@@ -128,13 +128,23 @@ fu_uefi_dbx_device_ensure_checksum(FuUefiDbxDevice *self, GError **error)
 
 	/* add the last checksum to the device */
 	sigs = fu_firmware_get_images(dbx);
-	if (sigs->len > 0) {
-		FuEfiSignature *sig = g_ptr_array_index(sigs, sigs->len - 1);
+
+	for (guint i = sigs->len; i > 0; i--) {
+		FuEfiSignature *sig = g_ptr_array_index(sigs, i - 1);
+		const gchar *owner = fu_efi_signature_get_owner(sig);
 		g_autofree gchar *csum =
 		    fu_firmware_get_checksum(FU_FIRMWARE(sig), G_CHECKSUM_SHA256, NULL);
+
+		/* Skip non-Microsoft owner GUIDs */
+		if (g_strcmp0(owner, FU_EFI_SIGNATURE_GUID_MICROSOFT) != 0) {
+			g_info("Skipping DBX. Not Microsoft: %s", owner);
```

Comment:
can we make this a g_debug please? Otherwise it's very verbose on HP machines that have a lot of non-MS entries too. Maybe somethign like `g_debug("skipping dbx entry %s as non-microsoft", csum);`